addons/stock/models/stock_picking.py

```diff
@@ -587,6 +587,11 @@ def create(self, vals):
 
         return res
 
+    def copy_data(self, default=None):
+        default = default or dict()
+        default['immediate_transfer'] = False
+        return super(Picking, self).copy_data(default)
```

Comment:
```suggestion
        return super().copy_data(default)
```

If I remember correctly in 14, you don't need to specify the parameters.